rclc/include/rclc/timer.h

```diff
@@ -40,11 +40,26 @@ extern "C"
  * \param[in] support the rclc_support_t object
  * \param[in] timeout_ns the time out in nanoseconds of the timer
  * \param[in] callback the callback of the timer
+ * \param[in] autostart the state of the timer at initialization
  * \return `RCL_RET_OK` if successful
  * \return `RCL_ERROR` (or other error code) if an error occurred
  */
 RCLC_PUBLIC
 rcl_ret_t
+rclc_timer_init_default2(
+  rcl_timer_t * timer,
+  rclc_support_t * support,
+  const uint64_t timeout_ns,
+  const rcl_timer_callback_t callback,
+  bool);
```

Comment:
```suggestion
  bool autostart);
```

like in https://github.com/ros2/rcl/blob/b55c41b8a877508f69a39daf8f0f8d9e551c69e8/rcl/include/rcl/timer.h#L172